module/VuFind/src/VuFind/Db/Entity/UserList.php

```diff
@@ -49,6 +49,13 @@ class UserList implements UserListEntityInterface
 {
     use DateTimeTrait;
 
+    /**
+     * Constant for default type of user list i.e favorites.
+     *
+     * @var string
+     */
+    public const TYPE_USERLIST = 'userlist';
```

Comment:
I wonder if we need a better name than 'userlist'. The entity is called 'UserList' so having 'userlist' as a type of 'UserList' seems potentially confusing. Would it be better to call these favorites? Or even to use 'default' as the default?